src/pages/index.js

```diff
@@ -90,7 +90,24 @@ const DescriptiveText = styled(P)`
     margin-bottom: ${size.medium};
 `;
 
-const IndexPageContent = ({ stream, title, twitchVideo, featuredArticles }) => {
+// TODO: Generalize as new content types are supported
+const FeaturedHomePageItem = ({ item }) => {
+    const itemType = item.type;
```

Comment:
Tiny nit, since we are only using `itemType` once here we probably don't need this variable. We can just use `item.type` directly in the conditon.

Reply:
Good catch! I was debating how I wanted to represent the item `type` before settling on an object property (which now makes me think more about adding TS in the future)